apps/console/src/features/identity-providers/constants/identity-provider-management-constants.ts

```diff
@@ -168,6 +168,11 @@ export class IdentityProviderManagementConstants {
         PROFILE: "profile"
     };
 
+    /**
+     * Google One Tap enabling request parameter
```

Comment:
Format to --> `Google One Tap enabling request parameter.`

Reply:
Fixed 9b4b2bc25c47eef914238e8c4592fd165da371c6